src/routing_table/mod.rs

```diff
@@ -300,9 +301,10 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
 
     pub fn is_in_our_group(&self, name: &T) -> bool {
         if self.our_group_prefix.matches(name) {
-            return unwrap!(self.groups.get(&self.our_group_prefix)).contains(name);
+            unwrap!(self.groups.get(&self.our_group_prefix)).contains(name)
```

Comment:
Instead of a _long_ `else` block, I'd also prefer an early `return`, but in this case I don't feel strongly about it. Happy to follow a guideline, if we can agree on one. :-D